### PR TITLE
Extract ctags support as a stand-alone module

### PR DESCRIPTION
**Pull request by markrtuttle**
This pull request extracts ctags support as a standalone module so it can be copied into other projects.  

Also modify ctags output to include the symbol kind (eg, "function").

Also advance version to 3.3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/cbmc_viewer/ctagst.py [hunk 1]
```diff
@@ -121,18 +129,20 @@ def exhuberant_tag(root, string):
...
     try:
         logging.info("Running legacy ctags")
-        stdout, _ = runt.popen(cmd, cwd=root)
+        stdout, _ = popen(cmd, cwd=root)
         strings = stdout.splitlines()
     except UserWarning:
         logging.info("Legacy ctags failed")
         strings = []
-
+    print(strings)
```
Comment (angelonakos) [inline src/cbmc_viewer/ctagst.py:145]: Should this be removed?

Comment (markrtuttle) [reply src/cbmc_viewer/ctagst.py]: Done.  Thanks!  Good catch.

**Review by angelonakos — APPROVED** [review]

**Comment by ronakfof** [comment]
Since you're using `coreHTTP` to test this, it uses `litani` submodule for which you need to install PIP dependencies (**Jinja2**) explicitly.
`brew install litani` installs pip packages only for the brew package in package's isolated env.

**Comment by ronakfof** [comment]
> Since you're using `coreHTTP` to test this, it uses `litani` submodule for which you need to install PIP dependencies (**Jinja2**) explicitly. `brew install litani` installs pip packages only for the brew package in package's isolated env.

Seems like you are already updating corehttp to litani installation. That works too.

**Pull request merged**